### PR TITLE
docs: document goldsky's Search endpoint

### PR DESCRIPTION
**Pull request by jeffling**
Hey! Added some documentation for the `arweave-search.goldsky.com` endpoint. Would love your feedback!

### CODE REVIEW

**Review by mrsaifullah52 — APPROVED** [review]
Looks good

**Comment by DanMacDonald** [comment]
Woo, this is great, love having the superset GraphQL features documented. 🥳

My feedback is mostly about word choice. "Gateways" are a bit of an overloaded term. The Arweave.net gateway is really a behemoth of several services that can be hosted and operated independently.  The term we've been using for "GraphQL Service" is "Indexing Service", the CDN that caches tx data "Transaction Data Service", there's also "Bundling Services", and "Transaction Sequencing" services.

We're starting to run into a proliferation of the word Gateway which is robbing it of any of its meaning.  Warp has the "Warp Gateway", ar-io operates the arweave.net gateway and decentralized gateways and now goldsky is operating a "Search Gateway".  The interesting thing to note here is that most of these disparate components can actually be composed to create something like the arweave.net gateway so calling them gateways feels like a misnomer. 

To bring these docs into alignment with the rest of the cookbook naming, it would be great to see the Goldsky service referred to an an "indexing service".  Thoughts?

Example in the diagram here. https://cookbook.arweave.dev/concepts/permawebApplications.html

**Comment by jeffling** [comment]
Great feedback, will tie that in. It's a bit different, because Goldsky support for arweave means we could actually mirror it into a search service, ML service, analytics warehouse etc so we tend to make the distinction between Indexing and Querying. But I definitely think we should go with the right community term here!

**Comment by jeffling** [comment]
> Great feedback, will tie that in. It's a bit different, because Goldsky support for arweave means we could actually mirror it into a search service, ML service, analytics warehouse etc so we tend to make the distinction between Indexing and Querying. But I definitely think we should go with the right community term here!

**Comment by DanMacDonald** [comment]
Perhaps calling it a "Search Service" leans in to the distinction here? It's better differentiated than "Search Gateway"

File: docs/src/guides/querying-arweave/search-indexing-service.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+The Search Gateway Syntax is less strict, and allows for searching just for the Tag name or value
+
+#### Examples
+Search for transactions with the tag value 'cat'
+
+```graphql:no-line-numbers
+query just_values {
+	transactions(
+		first: 10,
```
Comment (rosmcmahon) [inline docs/src/guides/querying-arweave/search-indexing-service.md:42]: tabs

**Comment by DanMacDonald** [comment]
Edits look good, the only other thing to add is a link to this doc from the README.md in the `guides` section.  Eventually we'll do something more with these landing pages, but for now they help with navigating the content on mobile.

https://github.com/twilson63/permaweb-cookbook/blob/main/docs/src/guides/README.md

File: docs/src/guides/querying-arweave/search-indexing-service.md (hunk 1 above)
Comment (jeffling) [reply docs/src/guides/querying-arweave/search-indexing-service.md]: fixed!

**Comment by jeffling** [comment]
@DanMacDonald added to readme!

**Pull request merged**